openage/convert/main.py

```diff
@@ -156,6 +156,26 @@ def convert_assets(assets, args, srcdir=None):
     conversion experience, then passes them to .driver.convert().
     """
 
+    assets = root["assets"]
+    cfg = root["cfg"]
+
+    asset_location_path = cfg / "asset_location"
+    if srcdir is None:
+        try:
+            with asset_location_path.open() as file_obj:
+                srcdir = CaseIgnoringDirectory(file_obj.read().strip()).root
+                print('Found previous AOE2 install path: ' + srcdir.resolve_native_path().decode('utf-8'))
+                while True:
+                    print("Should we use this directory? [Y/n]")
+                    user_selection = input("> ")
+                    if user_selection.lower() in {"yes", "y", ""}:
+                        break
+                    elif user_selection.lower() in {"no", "n"}:
+                        srcdir = None
+                        break
```

Comment:
maybe we should add an `else` with a "wtf don't understand"-like message :)

Reply:
fixed